bcManager/bcManager.py

```diff
@@ -880,17 +1554,76 @@ async def get_stats_updates_channel(self, guild: discord.Guild):
 
         return await ii_cat.create_text_channel(STATS_UPDATES_CHANNLE, sync=True)
 
+    async def get_match_tier_role_and_emoji_url(self, ctx, match):
+        if match['report'].get('winner'):
+            franchise_role, tier_role = await self.team_manager_cog._roles_for_team(ctx, match['report']['winner'])
+            emoji_url = (await self.team_manager_cog._get_franchise_emoji(ctx, franchise_role)).url
+        else:
+            franchise_role, tier_role = await self.team_manager_cog._roles_for_team(ctx, match['home'])
+            emoji_url = ctx.guild.icon_url
+        
+        return tier_role, emoji_url
+
+    async def assign_ff_reactions(self, message, deep_report):
+        for react in deep_report['ff_able_reacts']:
+            await message.add_reaction(react)
+
+    def get_select_reaction(self, int_or_hex):
+        try:
+            if type(int_or_hex) == int:
+                return struct.pack('<I', int_or_hex).decode('utf-32le')
+            if type(int_or_hex) == str:
+                return struct.pack('<I', int(int_or_hex, base=16)).decode('utf-32le') # i == react_hex
+        except:
+            return None
+
     def is_captain(self, player):
         for role in player.roles:
             if role.name.lower() == "captain":
                 return True
         return False
-        
+    
+    def get_latest_account_replay_by_plat_id(self, guild, platform, plat_id):        
+        bapi : ballchasing.Api = self.ballchasing_api[guild]
+        data = bapi.get_replays(player_id=f"{platform}:{plat_id}", sort_by="replay-date", sort_dir="desc", count=1)
+        return next(data, {})
+
+    def get_player_data_from_replay_by_plat_id(self, replay_json, platform, platform_id):
+        for team in ['blue', 'orange']:
+            for player in replay_json[team].get('players', []):
+                account_match = (
+                    player.get("id", {}).get('platform', None) == platform
+                    and
+                    player.get("id", {}).get('id', None) == platform_id
+                )
+                if account_match:
+                    return player
+        return {}
+    
+    def get_latest_player_data_by_platform_name(self, guild, platform, plat_name):        
+        bapi : ballchasing.Api = self.ballchasing_api[guild]
+        rsc_steam_id = "76561199096013422"
```

Comment:
what's this magic number, should it be in code vs as an env variable or..?

Reply:
That's my personal discord id, I need to update that out as well.

Reply:
lol alright the above is actually incorrect. I believe that's exactly what its named: That is the Steam ID for the RSC account associated with the league.

I'm gonna leave it in for now. We should probably have it be a configurable value in the future.